DOCS/man/mpv.rst

```diff
@@ -431,7 +428,7 @@ tree and play the longest title.
     a bitmap video stream which can be superimposed over the main
     movie. mpv's subtitle styling and positioning options and keyboard
     shortcuts generally do not work with image-based subtitles.
-    Exceptions include options like ``--stretch-dvd-subs`` and 
+    Exceptions include options like ``--stretch-dvd-subs`` and
     ``--stretch-image-subs-to-screen``.
```

Comment:
just thought it would be better to remove the trailing whitespace in this commit instead of having a separate commit just to remove it.